NewFEUBot.py

```diff
@@ -0,0 +1,79 @@
+import discord

```

Comment:
What is the difference between this file and `feubot.py`? If this is intended to be the main entry point for the program, why not add it to `feubot.py` directly?

Reply:
To answer your question, github throws a tantrum if I try to do anything with this file and the old memes, reactions and helpful files.

Reply:
That's why it wouldn't merge before. It said there was a complex conflict that github could not resolve.

Reply:
If that's the case, I'd like to keep `feubot.py` as the entry point to this program. I don't know what you're doing to make these commits, so I can't offer any advice on that front.